browser/browsing_data/counters/brave_site_settings_counter_unittest.cc

```diff
@@ -46,7 +46,7 @@ class BraveSiteSettingsCounterTest : public testing::Test {
     auto* zoom_map =
         content::HostZoomMap::GetDefaultForBrowserContext(profile());
 #else
-    auto* zoom_map = nullptr;
+    content::HostZoomMap* zoom_map = nullptr;
```

Comment:
Otherwise there is error
`../../brave/browser/browsing_data/counters/brave_site_settings_counter_unittest.cc:49:11: error: variable 'zoom_map' with type 'auto *' has incompatible initializer of type 'nullptr_t'
    auto* zoom_map = nullptr;`